cli.js

```diff
@@ -19,30 +43,78 @@ const debouncer = debounce(cb => cb(), 200);
 
 const cli = meow(`
 	Usage
-	  $ emoj [text]
+	  $ emoj [OPTIONS] [text]
 
 	Example
 	  $ emoj 'i love unicorns'
 	  🦄  🎠  🐴  🐎  ❤  ✨  🌈
 
 	Options
-	  --copy -c  Copy the first emoji to the clipboard
+	  --copy -c [choice]  Copy the emoji on index <choice> to the clipboard
```

Comment:
First of all: Sorry for the late reply.
Second: This is a valid concern, although I have to say, that not knowing which index you want also includes the first ony, which is selected by default. This way you can first do `emoj horsie` and then select the one you want with `emoj horsie --copy 3` (for example). I thought this allows users to explicitly select an indexed emojie, if it's not the first one you get via `emoj` on itself.